src/src/com/microsoft/aad/adal/Oauth2.java

```diff
@@ -579,7 +579,6 @@ public String encodeProtocolState() {
      */
     private AuthenticationResult processTokenResponse(HttpWebResponse webResponse) {
         AuthenticationResult result = new AuthenticationResult();
```

Comment:
nit: consider having it only declared, but not initialized. This way you'll ensure that it's set correctly below and you aren't missing a case